src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -1162,7 +1165,7 @@ public LambdaInfo WithMatch(Match<SyntaxNode> match, SyntaxNode newLambdaBody)
 
                     // exception handling around the statement:
                     CalculateExceptionRegionsAroundActiveStatement(
```

Comment:
>CalculateExceptionRegionsAroundActiveStatement [](http://example.com/codeflow?start=20&length=46)

If we don't have `match` then that means the lambda containing the AS was removed (or we couldn't find it). If that is the case the exception region analysis does not make much sense, so I think we should skip it.